mlx/backend/cuda/gemms/cublas_gemm.cpp

```diff
@@ -0,0 +1,290 @@
+// Copyright © 2025 Apple Inc.
+
+#include "mlx/backend/cuda/gemms/cublas_gemm.h"
+#include "mlx/backend/cuda/device.h"
+#include "mlx/dtype_utils.h"
+#include "mlx/utils.h"
+
+#include <fmt/format.h>
+
+void check_cublas_error(const char* name, cublasStatus_t err) {
```

Comment:
This is no longer needed, it has been moved to `mlx/backend/cuda/utils.h` by https://github.com/ml-explore/mlx/pull/2385.